.github/workflows/publish.yml

```diff
@@ -15,9 +15,17 @@ jobs:
           rust-version: stable
       - name: Check release validity
         run: sh .github/scripts/check-release.sh
-      - name: Build
+      - name: Build meilisearch crate
         run: cargo build --release
+      - name: Build meilisearch crate
+        run: |-
+          cd meilisearch-index-setting-macro
```

Comment:
Why this name of directory? `meilisearch-index-setting-macro`

Reply:
It was set like that by the contributor that created the macro. You don't like it?